tests/config/test_tls.py

```diff
@@ -176,19 +183,21 @@ def test_whitelist_idna_result(self):
         t = TestConfig()
         t.tls.read_config(config, config_dir_path="", data_dir_path="")
 
-        cf = FederationPolicyForHTTPS(t)
+        cf = FederationPolicyForHTTPS(cast(HomeServerConfig, t))
 
         # Not in the whitelist
         opts = cf.get_options(b"notexample.com")
+        assert isinstance(opts, SSLClientConnectionCreator)
```

Comment:
`_verifier` isn't part of the interface, but we know the underlying implementation.